middleware/initKeycloak.ts

```diff
@@ -9,7 +9,13 @@ import { isSignoutRoute } from '@/utils'
 export default async function ({ $config, route }) {
   console.info('Starting Keycloak service...')
 
-  await KeycloakService.setKeycloakConfigUrl($config.keycloakConfigPath)
+  const keycloakConfig: any = {
+    url: $config.keycloakAuthUrl,
+    realm: $config.keycloakRealm,
+    clientId: $config.keycloakClientId
+  }
```

Comment:
Should the other UI projects initialize these new fields? Please create tickets if needed.

Reply:
We will do the same way for other projects when we migrate them to firebase.

Reply:
Sounds good.